src/com/epam/javacourse/homework4/GreenhouseManagement.java

```diff
@@ -0,0 +1,118 @@
+package com.epam.javacourse.homework4;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GreenhouseManagement implements Greenhouse {
+
+    public final List<Plant> plants;
+    public double temperature;
+
+    public GreenhouseManagement() {
+        this.plants = new ArrayList<>();
+        this.temperature = 0;
+    }
+
+    public void addPlant(Plant plant) {
+        if (!plants.contains(plant)) {
+            this.plants.add(plant);
+            System.out.println("Plant " + plant.getPlantName() + " added to the greenhouse.");
+        } else {
+            System.out.println("Plant " + plant.getPlantName() + " is already in the greenhouse.");
+        }
+    }
+
+    @Override
+    public void createPlant(String plantName, String plantType, String nativeRegion) {
```

Comment:
This method is never used and can be deleted.

Reply:
Done